admin/class-metabox.php

```diff
@@ -1018,7 +1027,7 @@ function column_content( $column_name, $post_id ) {
 				}
 			}
 
-			echo '<div title="', esc_attr( $title ), '" class="wpseo-score-icon ', esc_attr( $score_label ), '"></div>';
+			echo '<div title="', esc_attr( $title ), '" id="wpseo-score-icon" class="wpseo-score-icon ', esc_attr( $score_label ), '"></div>';
```

Comment:
With no textual content, this is going to be an empty column for screen readers. Adding an aria-label _may_ help here.